src/content/docs/Language Overview/examples.md

```diff
@@ -213,7 +213,7 @@ fn void! test(int x)
     defer try io::printn("X");
     defer catch io::printn("B");
     defer catch (err) io::printfn("%s", err.message);
-    if (x == 1) return FooError!;
```

Comment:
In a way, this typo supports [the point I was trying to make in Discord](https://discord.com/channels/650345951868747808/820761627967488040/1292196408350343299) about `FooError!` being a more natural notation for producing errors than `FooError?`.